java.js

```diff
@@ -74,8 +87,31 @@ Form.addEventListener('submit', (e) => {
   Book.addBook(book);
   Book.clearField();
 });
+list.addEventListener('click', () => {
+  nav(li, add, contact);
+});
 
+addnew.addEventListener('click', () => {
+  nav(add, li, contact);
+});
+
+contac.addEventListener('click', () => {
+  nav(contact, add, li);
+});
+function date(timing) {
+  const dee = new Date();
+  let hours = dee.getHours();
+  const ampm = hours >= 12 ? 'pm' : 'am';
+  hours %= 12;
+  const mont = [
+    'january', 'feburary', 'march',
+    'april', 'may', 'june', 'july', 'august', 'september', 'october', 'november', 'december'];
+  const display = `${mont[dee.getMonth()]}
+  ${dee.getDate()}th ${dee.getFullYear()}, ${hours}:${dee.getMinutes()}${ampm}`;
+  timing.innerText = display;
+}
```

Comment:
- [ OPTIONAL ] I highly recommend making the UI styles here as shown in the given images so please add some more styles for the displyed books to make it looks better and to improve from user experince (UX) 🙏🏻💯🔥🔥


- Your design
![image](https://user-images.githubusercontent.com/58553711/212444988-eee1d6d3-9bb0-4173-80d6-3edba25d8bb8.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/212445003-4f2ba649-01cd-4762-9c69-b0d0aec23c5c.png)